### PR TITLE
Ci/fix helm publish

### PR DESCRIPTION
**Pull request by acellard**

### CODE REVIEW

**Review by alexisdeh — APPROVED** [review]

File: .github/workflows/helm.yaml [hunk 1]
```diff
@@ -13,13 +13,28 @@ jobs:
...
-            - uses: actions/checkout@v2
+            - uses: actions/checkout@v3
             - uses: actions/setup-python@v2
               with:
                   python-version: '>=3.9.0 <4.0.0'
             - run: python3 -m pip install -r ci/cilib/requirements.txt
 
             - run: helm lint charts/substra-frontend
+
+            - uses: azure/setup-helm@v3.3
```
Comment (mblottiere) [inline .github/workflows/helm.yaml:24]: `helm lint` uses lint, but maybe from the distro and not the desired version?

File: .github/workflows/helm.yaml [hunk 2]
```diff
@@ -13,13 +13,28 @@ jobs:
         name: Package Helm chart
         runs-on: ubuntu-latest
         steps:
-            - uses: actions/checkout@v2
+            - uses: actions/checkout@v3
             - uses: actions/setup-python@v2
               with:
                   python-version: '>=3.9.0 <4.0.0'
             - run: python3 -m pip install -r ci/cilib/requirements.txt
```
Comment (mblottiere) [inline .github/workflows/helm.yaml:20]: This seems unnecessary now

File: .github/workflows/helm.yaml [hunk 3]
```diff
@@ -13,13 +13,28 @@ jobs:
...
+
+            - uses: azure/setup-helm@v3.3
+              with:
+                  version: 'v3.5.0'
+              id: install
+
+            - name: Add dependencies repo
+              run: |
+                  helm repo add bitnami https://charts.bitnami.com/bitnami
+
```
Comment (mblottiere) [inline .github/workflows/helm.yaml:32]: The chart doesn't have dependencies

File: .github/workflows/helm.yaml [hunk 4]
```diff
@@ -13,13 +13,28 @@ jobs:
...
+            - name: Package chart
+              run: |
+                  helm dep build charts/substra-frontend
```
Comment (mblottiere) [inline .github/workflows/helm.yaml:35]: No dependencies:
```suggestion
```

File: .github/workflows/helm.yaml (hunk 2 above)
Comment (acellard) [reply .github/workflows/helm.yaml]: Removed

File: .github/workflows/helm.yaml (hunk 3 above)
Comment (acellard) [reply .github/workflows/helm.yaml]: Removed, thanks

File: .github/workflows/helm.yaml (hunk 4 above)
Comment (acellard) [reply .github/workflows/helm.yaml]: Removed

File: .github/workflows/helm.yaml (hunk 1 above)
Comment (acellard) [reply .github/workflows/helm.yaml:17]: Moved the install before the lint. It's better to all use the same version, right ?

Comment (mblottiere) [reply .github/workflows/helm.yaml:17]: Yes 👍

**Review by mblottiere — APPROVED** [review]

**Pull request merged**